docker-compose.yml

```diff
@@ -23,6 +23,11 @@ services:
       discovery.type: single-node
     ports:
       - "9200:9200"
+    networks:
+      potsie:
+        aliases:
+          - elasticsearch
+      default:
```

Comment:
What does an empty default do ? Just curious)

Reply:
It defaults to defaults :stuck_out_tongue: